k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.h

```diff
@@ -86,6 +89,12 @@ namespace LCIO2EDM4hepConv {
    */
   podio::Frame convertEvent(EVENT::LCEvent* evt);
 
+  /*
+  * Putting all the parameters of the event into the podio frame 
+  */
+  void convertEventParameters(EVENT::LCEvent* evt,podio::Frame*event);
```

Comment:
```suggestion
  template <typename TypeWithParams>
  void convertParameters(const TypeWithParams* lcioVal, podio::Frame& frame);
```
From the looks of the implementation, this should work for anything in LCIO that has a `getParameters` method so we can make this very generic and adjust the naming of the variables a bit to make that clear.

Additionally we can pass the `Frame` as a reference, since we don't use it as a pointer anywhere else. No need to start now ;)